utils/capellacontrol/controller.go

```diff
@@ -555,6 +555,53 @@ type CreateClusterRequest_Spec struct {
 	Services        []string                              `json:"services"`
 }
 
+type CreateDeployClusterRequest struct {
+	CIDR        string                           `json:"cidr"`
+	Description string                           `json:"description"`
+	Name        string                           `json:"name"`
+	Package     string                           `json:"package"`
+	TenantId    string                           `json:"tenantId"`
+	ProjectId   string                           `json:"projectId"`
+	Provider    string                           `json:"provider"`
+	Region      string                           `json:"region"`
+	Server      string                           `json:"server"`
+	Override    CreateOverrideRequest            `json:"overRide,omitempty"`
+	SingleAZ    bool                             `json:"singleAZ"`
+	Specs       []CreateDeployClusterRequestSpec `json:"specs"`
+	Timezone    string                           `json:"supportTimezone"`
+}
+
+type CreateDeployClusterRequestSpec struct {
+	Compute         CreateCompute                         `json:"compute"`
+	Count           int                                   `json:"count"`
+	Disk            CreateClusterRequest_Spec_Disk        `json:"disk"`
+	DiskAutoScaling CreateClusterRequest_Spec_DiskScaling `json:"diskAutoScaling"`
+	Services        []CreateServices                      `json:"services"`
+}
+
+type CreateCompute struct {
```

Comment:
Similar to above, this should be `DeployClusterRequest_Spec_Compute` I think.  All of these structures should follow the existing style.